iuf

```diff
@@ -436,8 +436,8 @@ def main():
 
     install_sp.add_argument("--limit-management-rollout", action="store", nargs='+',
         default=["Management_Worker"],
-        help="""Override list used to target specific role_subrole(s) only when rolling out management nodes.""")
-
+        help="""Override list used to target specific role_subrole(s) only when rolling out management nodes.
+        Defaults to the Management_Worker role.""")
```

Comment:
Unrelated, but can you change the default for --limit-managed-rollout to be "Compute" and update the help section to refelect that?

Reply:
Sure, done